tcms/kiwi_auth/tests/test_views.py

```diff
@@ -195,28 +196,28 @@ def test_register_user_by_email_confirmation(self, send_mail):
             "site_domain": site.domain,
             "confirm_url": confirm_url,
         }
-        expected_subject = (
-            settings.EMAIL_SUBJECT_PREFIX
-            + _("Your new %s account confirmation") % site.domain
-        )
-        expected_body = (
-            _(
-                """Welcome %(user)s,
-thank you for signing up for an %(site_domain)s account!
-
-To activate your account, click this link:
-%(confirm_url)s"""
+        with override(settings.LANGUAGE_CODE):
```

Comment:
I used override here too to use language code defined in settings.
Should I remove translation from the `expected_subject` and `expected_body`?
The tests might fail then if we change `LANGUAGE_CODE` in settings

Reply:
This shouldn't fail. `mailto` will send based on the language coming from settings and that should be the same language in which we run the test suite anyway. IMO this should be passing even without any changes.

Reply:
`mailto` will change the subject and body but I use override here to change `expected_subject` and expected_body`

Reply:
Sure, but the thing here is that during non-English tests we modify the server settings and client and server languages should be the same. Which should mean that even without override the gettext function (`_`) should result in exactly the same translation. Is it not doing that ?

Reply:
> during non-English tests we modify the server settings and client and server languages should be the same

Sorry, my bad. While testing locally, I was modifying client language only. I have removed `override` now.